packages/calling/src/Metrics/index.ts

```diff
@@ -321,6 +321,43 @@ class MetricManager implements IMetricManager {
       this.webex.internal.metrics.submitClientMetrics(name, data);
     }
   }
+
+  public submitBNRMetric(
+    name: METRIC_EVENT,
+    metricAction: string,
```

Comment:
Should this be the new enum created for the metrics?

Reply:
done